### DISCUSSION TITLE
Multiple windows

### DISCUSSION THREAD

**Discussion by katiereiter** [rxhanson/RectanglePro-Community#175] [Q&A] [answered]
What are 2x2/3 Tiles and Tidy supposed to do? They move my windows, but not in a predictable way

**Comment by rxhanson** [comment, marked as answer]
The 2x2 & 2x3 tiles ones just take the top 4 or 6 windows, respectively, and tile them in a grid. Useful only in fairly specific scenarios, and there aren't guarantees on which windows will be in which specific location.

Tidy will snap windows in a snap location that is nearest to the current size/position of the window. For example, if you have a finder window that is closest to the bottom right corner but it's not quite a quarter of the screen, then it'll get snapped into a quarter. Again, this is a pretty specific use case - one where you manually resize and move windows around quite a bit. I don't use it because I always snap my windows with throws and only move/resize sparingly when I don't care about the windows being snapped in a specific location, but there are people that like the Tidy.

**Reply by katiereiter** [reply]
Ahh makes sense. Thanks! :)
